.github/workflows/test.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        sdk: [dev, beta]
+        sdk: [dev, stable]
```

Comment:
ideally, this would be the oldest supported SDK. So `2.12.0`?

Reply:
Well, ideally, it would be all supported SDKs and versions of everything.   Yet, it seems like a bad idea to only test the oldest one and not test against the most current which presumably would be the version most likely to be used widely, eventually.